components/builder-notify/src/main.rs

```diff
@@ -1,21 +1,27 @@
 use habitat_builder_notify::{cli::Notify,
-                             error::Error,
                              server};
+use std::{fmt,
+          process};
 use structopt::StructOpt;
 
-#[macro_use]
-extern crate log;
 extern crate env_logger;
 
-fn main() -> Result<(), Error> {
+#[actix_rt::main]
+async fn main() {
     env_logger::init();
     match Notify::from_args() {
         Notify::Run { config } => {
-            if let Err(e) = server::run(config) {
-                error!("{}", e);
-                std::process::exit(1)
+            match server::run(config).await {
```

Comment:
curious why you changed this from the former code? Seems like you just need to add the `await` and maybe some textual detail to the `error!`. At the very least you would want the `println!` below to be `eprintln!` to ensure it is written to stderr.

Reply:
I see the similar logic applied to builder-api component.

https://github.com/habitat-sh/builder/blob/5e2955c29dd3771ec9ab562074cb3c700ef31cc7/components/builder-api/src/main.rs#L35

Reply:
hmm. well thats gonna print the error to stdout. I personally like the former and more succinct code. For instance there is no need to explicitly call `exit(0)`. Its really not a huge deal but if you are not going to use `error!` then `println!` should definitely be changed to `eprintln!`.